### PR TITLE
Add support for download URL overrides

### PR DESCRIPTION
**Pull request by akitada**
### Description

Add support for download URL overrides

### Issues Resolved

N/A

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by tas50** [comment]
We'll probably want to get a test suite setup to test this update scenario

**Comment by akitada** [comment]
@tas50 I've added a test suite but it needs new `mixlib-install` to run.
`chef_gem` part needs t o be updated to fetch the new version.

File: resources/updater.rb [hunk 1]
```diff
@@ -39,11 +41,10 @@
 
 action_class.class_eval do
   def load_mixlib_install
-    require 'mixlib/install'
```
Comment (tas50) [inline resources/updater.rb]: So we can still just do the require and rescue it which avoids adding the gem resource on every converge. Check out how we used to do it in aws

https://github.com/chef-cookbooks/aws/commit/11a13022b3d25cb016e85e91c9c38906c0e0af49#diff-6d3244c03f3201830fac92ae7572037bL20

Comment (akitada) [inline resources/updater.rb]: @tas Thank you for your feedback. I took your advice and updated the code.

**Comment by tas50** [comment]
Looks good. Thanks for working this from start to finish

**Pull request merged**